resources/web/template.html

```diff
@@ -161,1115 +101,314 @@
-								</ul>
-							</div>
-							<h3>Clients &amp; Integrations</h3>
-							<div class="itemizedlist">
-								<ul>
-									<li>
-										<a href="en/elasticsearch/client/java-api/current/index.html" target="_top">Java API — 1.4</a> —  
-										<a href="en/elasticsearch/client/java-api/index.html" target="_top">other versions</a>
-									</li>
-									<li>
-										<a href="en/elasticsearch/client/javascript-api/current/index.html" target="_top">JavaScript API — 3.1</a> —  
-										<a href="en/elasticsearch/client/javascript-api/index.html" target="_top">other versions</a>
-									</li>
-									<li>
-										<a href="en/elasticsearch/client/groovy-api/current/index.html" target="_top">Groovy API — 1.4</a> —  
-										<a href="en/elasticsearch/client/groovy-api/index.html" target="_top">other versions</a>
-									</li>
-									<li>
-										<a href="en/elasticsearch/client/net-api/current/index.html" target="_top">.NET API</a>
-									</li>
-									<li>
-										<a href="en/elasticsearch/client/php-api/current/index.html" target="_top">PHP API — 1.0</a> —  
-										<a href="en/elasticsearch/client/php-api/index.html" target="_top">other versions</a>
-									</li>
-									<li>
-										<a href="en/elasticsearch/client/perl-api/current/index.html" target="_top">Perl API</a>
-									</li>
-									<li>
-										<a href="en/elasticsearch/client/python-api/current/index.html" target="_top">Python API</a>
-									</li>
-									<li>
-										<a href="en/elasticsearch/client/ruby-api/current/index.html" target="_top">Ruby API</a>
-									</li>
-									<li>
-										<a href="en/elasticsearch/rivers/current/index.html" target="_top">Rivers — 1.4</a> —  
-										<a href="en/elasticsearch/rivers/index.html" target="_top">other versions</a>
-									</li>
-									<li>
-										<a href="en/elasticsearch/client/community/current/index.html" target="_top">Community contributions</a>
-									</li>
-								</ul>
-							</div>							
-						</div>
-					</div>
-				</div>
-				<!-- end body -->
-				<div class="col-xs-12 col-sm-4 col-md-4" id="right_col">
-					<div id="rtpcontainer" style="display: block;">
-						<div class="mktg-promo">
-						<h3>Getting Started Videos</h3>
-						<ul class="icons">
-							<li class="icon-elasticsearch-white"><a href="https://www.elastic.co/webinars/getting-started-elasticsearch?baymax=default&elektra=docs&storm=top-video">Starting Elasticsearch</a></li>
-							<li class="icon-kibana-white"><a href="https://www.elastic.co/webinars/getting-started-kibana?baymax=default&elektra=docs&storm=top-video">Introduction to Kibana</a></li>
-							<li class="icon-logstash-white"><a href="https://www.elastic.co/webinars/getting-started-logstash?baymax=default&elektra=docs&storm=top-video">Logstash Starter Guide</a></li>
-						</ul>
-						</div>
-					</div>
-				</div>
-			</div>
-		</div>
-	</section>
+          <!-- <div class="pageheader">
```

Comment:
Is there a good reason we should keep this? If it is commented out I think maybe we should trash it too

Reply:
++ deleted in `26ca0956f26`